test/runner/steps_test.js

```diff
@@ -0,0 +1,38 @@
+const expect = require('expect');
+const path = require('path');
+const exec = require('child_process').exec;
+
+const runner = path.join(__dirname, '/../../bin/codecept.js');
+const codecept_dir = path.join(__dirname, '/../data/sandbox/configs/steps');
+const codecept_run = `${runner} run`;
+const config_run_config = (config, grep) => `${codecept_run} --config ${codecept_dir}/${config} ${grep ? `--grep "${grep}"` : ''}`;
+
+describe('CodeceptJS Steps', () => {
+  it('should stop test, when step exceeded', (done) => {
```

Comment:
I"m just curious if those tests can ben written in async/await approach?

Reply:
yes, it can be.